clang/include/clang/Parse/Parser.h

```diff
@@ -3368,6 +3368,16 @@ class Parser : public CodeCompletionHandler {
   /// metadirective and therefore ends on the closing paren.
   StmtResult ParseOpenMPDeclarativeOrExecutableDirective(
       ParsedStmtContext StmtCtx, bool ReadDirectiveWithinMetadirective = false);
+  /// Parses target directive with thread_limit clause
+  ///
+  ///
+  void ParseOpenMPTargetThreadLimit(StmtResult & Directive, OpenMPDirectiveKind & DKind,
+       DeclarationNameInfo & DirName, SourceLocation & Loc, SourceLocation & EndLoc, unsigned & ScopeFlags,
```

Comment:
Why the additional space before the formal parameter name?

```suggestion
  void ParseOpenMPTargetThreadLimit(StmtResult &Directive, OpenMPDirectiveKind &DKind,
       DeclarationNameInfo &DirName, SourceLocation &Loc, SourceLocation &EndLoc, unsigned &ScopeFlags,
```